completions/ipsec

```diff
@@ -14,7 +14,7 @@ _ipsec_connections()
         COMPREPLY=($(compgen -W '"${COMPREPLY[@]}"' -- "$cur"))
 }
 
-_ipsec_freeswan()
+_comp_cmd_ipsec__freeswan()
```

Comment:
This is in a sense "abusing" the utility namespace for a completion function, ditto the strongswan variant below. I think that's ok.

Reply:
This is in line with https://github.com/scop/bash-completion/pull/883#discussion_r1174530020